packages/@expo/cli/src/lint/ESlintPrerequisite.ts

```diff
@@ -112,7 +114,6 @@ export class ESLintProjectPrerequisite extends ProjectPrerequisite<boolean> {
         requiredPackages: [
           { version: '^8.57.0', pkg: 'eslint', file: 'eslint/package.json', dev: true },
           {
-            version: '^7.0.0',
```

Comment:
This was required because `7.x.x` was published as `next` or `beta` instead of `latest`. Good to remove it